README.md

```diff
@@ -105,6 +118,8 @@ export default function runQuery(id) {
 
 Check out the [detailed API documentation](API.md#new-connectionpoolpartyconfig).
 
+You'll also want to familiarize yourself with [`node-mssql`'s documentation](https://github.com/tediousjs/node-mssql/blob/7f374a8d73b00b17aa5b5ea5621c4314fc6e2daa/README.md#documentation), as much of it applies to mssql-pool-party.
```

Comment:
should we be linking to master here, or are we going to keep it pinned?

Reply:
hmm, i suppose we should point it at master